cake-recipe.txt

```diff
@@ -7,7 +7,7 @@ Ingredients:
  - 1 teaspoon salt
  - 2 eggs
  - 1 cup milk
- - 1/2 cup vegetable oil
+ - 1/2 cup Canola oil
```

Comment:
I don't think Canola needs to be capitalized.  (I know it stood for Canadian Oil ... but I don't think it's trademarked.)